src/service/user.js

```diff
@@ -0,0 +1,53 @@
+class User {
+    constructor() {
+      this.users = new Map();
+      this.id = 0;
+    }
+  
+    getAllUsers() {
+      return this.users.values();
+    }
+  
+    createUser(user) {
+      this.id += 1;
+      this.users.set(this.id, { ...user, id: this.id });
+      return this.users.get(this.id);
+    }
+  
+    updateUser(id, role, name, email) {
+      if (this.getUser(id)){
+        const user = this.users.get(Number(id));
+        const  message = 'Successfully Updated';
+
+        this.users.set(Number(id), {
+          ...user, role, name, email, message
+        });
+        return this.users.get(Number(id));
+      }
+      else{
+        const  message = 'Failed Updated';
```

Comment:
can we have variable name as failMessage , successMessage also update messages like this "Failed to update",

Also check naming convention of a message

Try to reduce variable